package.json

```diff
@@ -39,7 +39,7 @@
     "@nextcloud/logger": "^1.1.2",
     "@nextcloud/moment": "^1.1.1",
     "@nextcloud/router": "^1.2.0",
-    "@nextcloud/vue": "^3.9.0",
+    "@nextcloud/vue": "^4.0.0-alpha.3",
```

Comment:
v4 will pull in `@nextcloud/router` v2 and that will drop some of the browsers that we still support or which are still supported for Nextcloud 20-21

Reply:
We decided to drop older browsers.